From d783144d6738af1d2ef3a91ef3cb2ef8529bb041 Mon Sep 17 00:00:00 2001
From: Florinel Iordache <florinel.iordache@nxp.com>
Date: Mon, 5 Nov 2018 17:04:05 +0200
Subject: [PATCH] arm64: dts: Added endianness information to dts serdes node

This change is required to specify that serdes HW peripheral is little-endian.

Signed-off-by: Florinel Iordache <florinel.iordache@nxp.com>
---
 arch/arm64/boot/dts/freescale/fsl-ls1088a.dtsi | 3 ++-
 arch/arm64/boot/dts/freescale/fsl-ls208xa.dtsi | 3 ++-
 arch/arm64/boot/dts/freescale/fsl-lx2160a.dtsi | 3 ++-
 3 files changed, 6 insertions(+), 3 deletions(-)

--- a/arch/arm64/boot/dts/freescale/fsl-ls1088a.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-ls1088a.dtsi
@@ -184,8 +184,9 @@
 		};
 
 		serdes1: serdes@1ea0000 {
-				reg = <0x0 0x1ea0000 0 0x00002000>;
 				compatible = "fsl,serdes-10g";
+				reg = <0x0 0x1ea0000 0 0x00002000>;
+				little-endian;
 		};
 
 		tmu: tmu@1f80000 {
--- a/arch/arm64/boot/dts/freescale/fsl-ls208xa.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-ls208xa.dtsi
@@ -833,8 +833,9 @@
 		};
 
 		serdes1: serdes@1ea0000 {
-				reg = <0x0 0x1ea0000 0 0x00002000>;
 				compatible = "fsl,serdes-10g";
+				reg = <0x0 0x1ea0000 0 0x00002000>;
+				little-endian;
 		};
 
 		ccn@4000000 {
--- a/arch/arm64/boot/dts/freescale/fsl-lx2160a.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-lx2160a.dtsi
@@ -581,8 +581,9 @@
 		};
 
 		serdes1: serdes@1ea0000 {
-				reg = <0x0 0x1ea0000 0 0x00002000>;
 				compatible = "fsl,serdes-28g";
+				reg = <0x0 0x1ea0000 0 0x00002000>;
+				little-endian;
 		};
 
 		i2c0: i2c@2000000 {
